tuning-crc-openshift-cluster/crc-perf-tuning.sh

```diff
@@ -0,0 +1,79 @@
+OC=${OC:-oc}
+
+######
+##  Series of steps to inject necessary ENV variables and resources related changes for CRC ##
+#####
+
+
+######
+##  Enable v1alpha1/settings API for using Podpresets to set ENV variables while pods get created ##
+#####
+echo 'Enable Kube V1/alpha API .....'
+./enable-alpha-api.sh
+./make-kube-control-manifests-immutable.sh
+sleep 60
```

Comment:
All the sleep statement should be removed from here and have logic in place which have info about for which resource/api/process we are waiting for.